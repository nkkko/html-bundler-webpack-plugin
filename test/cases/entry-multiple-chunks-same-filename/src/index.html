<!DOCTYPE html>
<html>
  <head>
    <!-- test: the same style file is defined in entry -->
    <link rel="stylesheet" href="./main.css">
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>