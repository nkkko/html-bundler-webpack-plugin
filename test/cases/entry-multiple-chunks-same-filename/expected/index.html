<!DOCTYPE html>
<html>
  <head>
    <!-- test: the same style file is defined in entry -->
    <link rel="stylesheet" href="css/main.90884336.css">
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>